packages/proto-signing/TEST_VECTORS.md

```diff
@@ -11,10 +11,10 @@ my branch (which can be rebased).
 ## Setup
 
 Test vectors are generated using those additions:
-https://github.com/CosmWasm/cosmos-sdk/pull/58
+https://github.com/cosmos/cosmos-sdk/pull/58
```

Comment:
Unfortunately I deleted the fork https://github.com/CosmWasm/cosmos-sdk a few weeks ago in a cleanup session. Our PR 58 is a very different one than PR 58 of the original repo.

Not sure what the bst change is here. The setup cannot easily be reproduced anymore.